internal/pkg/agent/cmd/run.go

```diff
@@ -133,6 +134,22 @@ func run(override cfgOverrider, testingMode bool, fleetInitTimeout time.Duration
 	defer cancel()
 	go service.ProcessWindowsControlEvents(stopBeat)
 
+	// detect otel
+	runAsOtel, err := otel.IsOtelConfig(ctx, paths.ConfigFile())
+	if err != nil {
+		return err
+	}
+
+	if runAsOtel {
+		return otel.Run(ctx, cancel, stop, testingMode)
+	}
+
+	// not otel continue as usual
+	return runElasticAgent(ctx, cancel, override, stop, testingMode, fleetInitTimeout, modifiers...)
+
+}
```

Comment:
Would really like to see otel integrated into the coordinator and runtime system. That way we get all the status reporting, and the ability to run both at the same time.

Reply:
@joshdover what was your idea of the scope for the initial integration?